assets/src/edit-story/utils/stripHTML.js

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export default function stripHTML(string) {
+  // @todo: implement a cheaper way to strip markup.
+  const buffer = document.createElement('div');
```

Comment:
Nit: a very quick improvement could be to just move `const buffer` out as a module constant. Not sure if we have any side-effect lint that would prevent us from doing that.

Reply:
Done in 580178f.